sentry_sdk/hub.py

```diff
@@ -327,7 +328,7 @@ def capture_event(
         event,  # type: Event
         hint=None,  # type: Optional[Hint]
         scope=None,  # type: Optional[Any]
-        **scope_args  # type: Any
+        **scope_args,  # type: Any
```

Comment:
We'll need to get rid of these trailing commas after `**kwargs` since they're throwing SyntaxErrors in <=3.5. I'm not sure why black suddenly started adding these. They officially dropped 3.5 support at the end of last year already.